packages/react-celo/src/connectors/wallet-connect.ts

```diff
@@ -47,7 +48,14 @@ export default class WalletConnectConnector
     readonly walletId?: string
   ) {
     super();
-    const wallet = new WalletConnectWalletV1(options);
+    const wallet = new WalletConnectWalletV1(
+      manualNetworkMode
+        ? {
+            init: options.init,
+            connect: { ...options.connect, chainId: undefined },
```

Comment:
What does passing `chainId: undefined` do exactly?

Reply:
it leaves the wallet on its current network rather than requesting it change it